apps/api/src/app/auth/services/passport/github.strategy.ts

```diff
@@ -6,7 +6,7 @@ import { AuthProviderEnum } from '@novu/shared';
 import { AuthService } from '../auth.service';
 
 @Injectable()
-export class GithubStrategy extends PassportStrategy(githubPassport.Strategy, 'github') {
+export class GitHubStrategy extends PassportStrategy(githubPassport.Strategy, 'github') {
```

Comment:
Sometimes exported items are APIs that can't be broken... dunno if that applies here.

Reply:
This is only used in: https://github.com/novuhq/novu/blob/2d3050a13b5dfc77b0d66e34ed2eac8a25299631/apps/api/src/app/auth/auth.module.ts
So renaming it there should be ok this change.